src/api/entities/MetadataEntry/index.ts

```diff
@@ -80,6 +85,15 @@ export class MetadataEntry extends Entity<UniqueIdentifiers, HumanReadable> {
    */
   public set: ProcedureMethod<SetMetadataParams, MetadataEntry>;
 
+  /**
+   * Removes the asset metadata value
+   *
+   * @throws
+   *   - if the Metadata doesn't exists
+   *   - if the Metadata value is locked
+   */
+  public clear: NoArgsProcedureMethod<void>;
```

Comment:
What about adding something like:

```suggestion
  public clear: NoArgsProcedureMethod<void>;
  
  public isModifiable(): Promise<{canModify: boolean; reason?: PolymeshError}> {
    // assertIsModifiable logic here
  }
```